firmware/application/ui/ui_geomap.cpp

```diff
@@ -613,7 +613,7 @@ void GeoMapView::focus() {
     geopos.focus();
 
     if (!geomap.map_file_opened())
-        nav_.display_modal("No map", "No world_map.bin file in\n/ADSB/ directory", ABORT);
+        nav_.display_modal("No map", "No world_map.bin file in\neither /SYS/ADSB/ or\n /USR/ADSB directory", ABORT);
```

Comment:
Should this say /RES/ADSB to match the .open above?

Reply:
ohhhh i'm so sorry, fixed.